crates/adapters/src/controller/mod.rs

```diff
@@ -447,21 +447,21 @@ impl Controller {
             -> Result<(Box<dyn DbspCircuitHandle>, Box<dyn CircuitCatalog>), ControllerError>,
     {
         let mut start: Option<Instant> = None;
-        let min_storage_rows = if controller.status.pipeline_config.global.storage {
+        let min_storage_bytes = if controller.status.pipeline_config.global.storage {
             // This reduces the files stored on disk to a reasonable number.
             controller
                 .status
                 .pipeline_config
                 .global
-                .min_storage_rows
-                .unwrap_or(1000)
+                .min_storage_bytes
+                .unwrap_or(1_000_000)
```

Comment:
How about we set this to 1 MiB instead?
```suggestion
                .unwrap_or(1024*1024)
```